crates/biome_html_parser/src/syntax/vue.rs

```diff
@@ -0,0 +1,140 @@
+use crate::parser::HtmlParser;
+use crate::syntax::parse_attribute_initializer;
+use crate::syntax::parse_error::expected_attribute;
+use crate::token_source::HtmlLexContext;
+use biome_html_syntax::HtmlSyntaxKind;
+use biome_html_syntax::HtmlSyntaxKind::*;
+use biome_html_syntax::T;
+use biome_parser::Parser;
+use biome_parser::parse_lists::ParseNodeList;
+use biome_parser::parse_recovery::ParseRecoveryTokenSet;
+use biome_parser::parsed_syntax::ParsedSyntax::{Absent, Present};
+use biome_parser::prelude::*;
+
+pub fn parse_vue_directive(p: &mut HtmlParser) -> ParsedSyntax {
+    let m = p.start();
+
+    p.bump_with_context(HTML_LITERAL, HtmlLexContext::InsideTag);
+    if p.at(T![:]) {
+        parse_vue_directive_argument(p).ok();
+    }
+    VueModifierList.parse_list(p);
+    if p.at(T![=]) {
+        parse_attribute_initializer(p).ok();
+    }
+
+    Present(m.complete(p, VUE_DIRECTIVE))
+}
+
+pub fn parse_vue_v_bind_shorthand_directive(p: &mut HtmlParser) -> ParsedSyntax {
+    if !p.at(T![:]) {
+        return Absent;
+    }
+
+    let m = p.start();
+
+    parse_vue_directive_argument(p).ok();
+    VueModifierList.parse_list(p);
```

Comment:
If `parse_vue_directive_argument` emits an error, it's almost possible that line 37 will fail too. Shouldn't we bail and emit a bogus node? Or are there test that shows that the parsing of modifiers can succeed?